composer/algorithms/alibi/alibi.py

```diff
@@ -29,32 +33,33 @@ def apply_alibi(
     mask_replacement_function: Union[Callable, None],
     optimizers: Optional[Optimizers] = None,
 ) -> None:
-    """Removes position embeddings and replaces the attention function and attention mask according to `AliBi.
+    """Removes position embeddings and replaces the attention function and attention mask
+    according as per :class:`~composer.algorithms.alibi.alibi.Alibi`.
 
-    <https://arxiv.org/abs/2108.12409>`_.
+    Example: Waiting on language model test fixtures
 
     Args:
-        model: model to transform
-        heads_per_layer: number of attention heads per layer
-        max_sequence_length: maximum sequence length that the
+        model (torch.nn.Module): model to transform
+        heads_per_layer (int): number of attention heads per layer
+        max_sequence_length (int): maximum sequence length that the
             model will be able to accept without returning an error
-        position_embedding_attribute: attribute for position
+        position_embedding_attribute (str): attribute for position
             embeddings. For example in HuggingFace's GPT2, the
             position embeddings are "transformer.wpe".
-        attention_module: module/class that will have its
+        attention_module (torch.nn.Module): module/class that will have its
             self-attention function replaced. For example, in
             HuggingFace's GPT, the self-attention module is
             transformers.models.gpt2.modeling_gpt2.GPT2Attention.
-        attr_to_replace: attribute that self-attention function will
+        attr_to_replace (str): attribute that self-attention function will
             replace. For example, in HuggingFace's GPT2, the
             self-attention function is "_attn".
-        alibi_attention: new self-attention function in which
+        alibi_attention (Callable): new self-attention function in which
             ALiBi is implemented. Used to replace
             "{attention_module}.{attr_to_replace}".
-        mask_replacement_function: function to replace model's
+        mask_replacement_function (Union[Callable, None]): function to replace model's
```

Comment:
Can you update the annotation to be `Optional[Callable[[torch.nn.Module, int], torch.nn.Module]]` (assuming that it is supposed to return a module? In addition, can you add what it means to leave it with the default value of ``None``?